db/migrate/20220824174034_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,8 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:users) do |t| 
+      t.confirmable 
```

Comment:
- [ ] I am getting this error while trying to migrate your database

![image](https://user-images.githubusercontent.com/62705947/186522669-1c01270f-43a7-4267-a118-f596ea9fdc2e.png)

Reply:
Hi @GzimAsani,
Everything looks good from my end, so I don't get why the migrations won't run on your computer. Thanks for your review though

Reply:
Hi @carlylechia, can you please try to clone your project from the beginning and try to run bundle and run migrations, and see if everything works?